Source/WebCore/platform/audio/gstreamer/AudioFileReaderGStreamer.cpp

```diff
@@ -133,6 +152,9 @@ AudioFileReader::~AudioFileReader()
 
     if (m_decodebin) {
         g_signal_handlers_disconnect_matched(m_decodebin.get(), G_SIGNAL_MATCH_DATA, 0, 0, nullptr, nullptr, this);
+#if PLATFORM(BCM_NEXUS)
+        g_signal_handlers_disconnect_matched(m_decodebin.get(), G_SIGNAL_MATCH_FUNC, 0, 0, nullptr, gpointer(decodebinAutoplugSelectCallback), nullptr);
```

Comment:
`gpointer(foo)` expands to `void*(foo)` iiuc, is this a valid cast?

Reply:
it seems to

Reply:
`g_signal_handlers_disconnect_matched` takes a `gpointer` as a parameter, so it's almost ok